tests/integration/compose/docker_compose_postgres.yml

```diff
@@ -1,7 +1,7 @@
 services:
     postgres1:
         image: postgres
-        command: ["postgres", "-c", "wal_level=logical", "-c", "max_replication_slots=4", "-c", "logging_collector=on", "-c", "log_directory=/postgres/logs", "-c", "log_statement=all", "-c", "max_connections=200"]
+        command: ["postgres", "-c", "wal_level=logical", "-c", "max_replication_slots=4", "-c", "logging_collector=on", "-c", "log_directory=/postgres/logs", "-c", "log_statement=all", "-c", "max_connections=200", "-c", "statement_timeout=180000"]
```

Comment:
There was a DROP DATABASE query of materialized pg database, which waited for a consume thread to finish, which stuck for exactly 30 minutes in a query to postgres, after those 30 minutes we received "connection to server failed: timeout expired error". This `statement_timeout` setting should help with this.

Reply:
Could you please add this as a comment?